src/app/pages/TopicPage/Curation/Subhead/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+
+import Link from './index.styled';
+
+import { LeftChevron, RightChevron } from '../../icons';
+
+import { ServiceContext } from '../../../../contexts/ServiceContext';
+
+interface Props {
+  children: React.ReactNode;
+  href?: string;
+}
+
+const Subhead = ({ children, href }: Props) => {
+  const { service, script, dir } = useContext(ServiceContext) as {
+    script: string;
+    service: string;
+    dir: string;
+  };
+  const Wrapper = href
+    ? ({ children: innerChildren }) => (
+        <Link href={href} service={service} script={script}>
+          <span>{innerChildren}</span>
+          {dir === 'ltr' ? <RightChevron /> : <LeftChevron />}
+        </Link>
+      )
+    : React.Fragment;
+  return (
+    <h2 id="content" tabIndex={-1}>
```

Comment:
There's probably some work needed here to get the `aria-labelledby` stuff implemented, but I think that is best done in the ticket that actually adds the multiple curations, because the elements we need aren't available yet.
```suggestion
    <h2>
```